### PR TITLE
DM-28727: Clean up pybind11 wrappers for afw.image

### PR DESCRIPTION
**Pull request by mwittgen**

### CODE REVIEW

File: python/lsst/afw/image/apCorrMap/_apCorrMap.cc [hunk 1]
```diff
@@ -67,7 +67,7 @@ void wrapApCorrMap(lsst::utils::python::WrapperCollection &wrappers) {
     });
 }
 
-PYBIND11_MODULE(apCorrMap, mod) {
+PYBIND11_MODULE(_apCorrMap, mod) {
```
Comment (TallJimbo) [inline python/lsst/afw/image/apCorrMap/_apCorrMap.cc:71]: I had forgotten that we had these separate subpackage builds in `afw.image` with their own `SConscript`s inside directories (`apCorrMap`, `exposure`, `image`, `maskedImage`, `pixel`).  With the exception of `pixel`, these are not intended to appear as public namespaces in Python (they aren't namespaces in C++), and I don't think there's any reason for them to be separate anymore.  Could you merge those packages (`apCorrMap`, `exposure`, `image`, `maskedImage`) into the `_imageLib` build and take them out of their subdirectories?

File: python/lsst/afw/image/image/_imageLib.cc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "lsst/utils/python.h"
+
+namespace lsst {
+namespace afw {
+namespace image {
+  //void wrapApCorrMap(lsst::utils::python::WrapperCollection &);
```
Comment (TallJimbo) [inline python/lsst/afw/image/image/_imageLib.cc]: Looks like a relic, but I think it'll just come back after you address my other comment.

**Review by TallJimbo — APPROVED** [review]
Looks good, but I'd like to go a step further and pull out some of the remaining subpackages, too (see in-line comment).

File: python/lsst/afw/image/apCorrMap/_apCorrMap.cc (hunk 1 above)
Comment (mwittgen) [reply python/lsst/afw/image/apCorrMap/_apCorrMap.cc:71]: I has some dependency problems when doing this. I would push this back and revisit when all other sub packages are using the pybind wrappers.

**Pull request merged**